quinn-proto/src/connection/assembler.rs

```diff
@@ -189,6 +189,22 @@ impl Assembler {
     }
 }
 
+/// A chunk of data from the receive stream
+#[non_exhaustive]
```

Comment:
What fields might we add to this in the future?

Reply:
I couldn't think of any; just being defensive, maybe overly so?

Reply:
My inclination is to reserve `non_exhaustive` for cases where we're pretty sure data will be added, at least so long as we have no reason to expect minor breakage to be disruptive.

Reply:
Removed it.